### PR TITLE
Added Admin Page

### PR DESCRIPTION
**Pull request by vijay-sb**
### This PR Introduces the Addition of the Admin Page
**Features Implemented**

**Temporary Swap:**  Functionality to temporarily swap timetable periods.
**Permanent Swap:**  Functionality to permanently swap timetable periods.
**Add Exams**:  Admin can add exam schedules.
**Add Assignments**:  Admin can add assignment details.
**Add Tutorial:**  Admin can add tutorial details
**Add Quiz:**  Admin can create quizzes.
**Add Evaluation:**  Admin can add evaluation schedules.
**Generate OTP:**  Feature to generate OTP
**Progress Bar:**  Visual indicator to Sem progress.
**Routing for Each Icon**:  Separate routes implemented for each endpoints

image

**TO DO**

- Build seprate pages to Implement the Functionality
- Styling Improvements

### CODE REVIEW

File: .expo/README.md
Comment (IAmRiteshKoushik) [inline .expo/README.md:1]: Please remove the `.expo` folder. This is system specific and generated by Expo. It ought not to be committed.

**Review by IAmRiteshKoushik — CHANGES_REQUESTED** [review]
We have a massive splurge of images. Are they all banners being used in some kind of rotation format or have they been downloaded and kept for testing purposes ? 

If they are only for testing purposes, then we should avoid bloating up the size of the repository with these many image assets at it may cause difficulty with further git-pulls to other developers due to the massive size.

**Comment by IAmRiteshKoushik** [comment]
Other than the above mentioned issues, a small suggestion to keep the file naming consistent. Any pages or components ought to be in `TitleCase` and other utilities / auxiliary functions can be in `lowercase`

File: .expo/README.md
Comment (vijay-sb) [reply .expo/README.md:1]: All the unused images and components have been removed

**Review by IAmRiteshKoushik — APPROVED** [review]
LGTM

File: chronos-app/components/AdminComponents/CustomFooter.tsx [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+import { View, Text, TouchableOpacity } from "react-native";
+import { Ionicons } from "@expo/vector-icons";
+
+const Footer = () => {
+  return (
+    <View className="bg-[#151515] px-6 py-8 rounded-t-3xl relative overflow-hidden border border-gray-700">
+      <View className="absolute inset-0">
+        <View className="absolute top-0 left-0 w-1/2 h-full bg-gradient-to-br from-[#1E293B] to-transparent opacity-20" />
+        <View className="absolute bottom-0 right-0 w-1/2 h-full bg-gradient-to-tl from-[#374151] to-transparent opacity-20" />
+      </View>
```
Comment (KiranRajeev-KV) [inline chronos-app/components/AdminComponents/CustomFooter.tsx:11]: What is this code used for?

**Review by KiranRajeev-KV — APPROVED** [review]
Looks good.

**Review by KiranRajeev-KV — APPROVED** [review]

**Pull request merged**